spec/controllers/api/current/blood_pressures_controller_spec.rb

```diff
@@ -52,6 +52,76 @@ def create_record_list(n, options = {})
         expect(patient.blood_pressures.count).to eq 10
         expect(response).to have_http_status(200)
       end
+
+      context 'recorded_at is sent' do
+        it 'sets the recorded_at sent in the params' do
+          recorded_at = 1.month.ago
+          blood_pressure = build_blood_pressure_payload(FactoryBot.build(:blood_pressure, recorded_at: recorded_at))
+
+          post(:sync_from_user, params: { blood_pressures: [blood_pressure] }, as: :json)
+
+          bp = BloodPressure.find(blood_pressure['id'])
+          expect(bp.recorded_at.to_i).to eq(recorded_at.to_i)
+        end
+
+        it 'does not modify the recorded_at for a patient' do
+          patient_recorded_at = 4.months.ago
+          patient = FactoryBot.create(:patient, recorded_at: patient_recorded_at)
+          older_bp_recording_date = 5.months.ago
+          blood_pressure = build_blood_pressure_payload(FactoryBot.build(:blood_pressure,
+                                                                         patient: patient,
+                                                                         recorded_at: older_bp_recording_date))
+          post(:sync_from_user, params: { blood_pressures: [blood_pressure] }, as: :json)
+
+          patient.reload
+          expect(patient.recorded_at.to_i).to eq(patient_recorded_at.to_i)
```

Comment:
`patient.recorded_at` should modify, right? Am I missing something?

Reply:
updated current should not modify the patient's recorded_at in case of v3.1 because they will sync the patient with updated `recorded_at`. We should update the description to say this.